source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -594,6 +596,225 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
 <quantity>30</quantity>
 ~~~
 
+
+### Size Variation Attributes
+
+Size variations is an enhancement to the current XML feed, that introduces new nodes (children)
+that include the information of size variations nested to the parent product.
+The parent product should be exactly the same with the current XML specification (including
+all fields, even those that are size variation specific e.g. quantity).
+
+The table below summarizes the size variation attributes recognized by
+<%= settings.site_name.capitalize %>:
+
+<div class="alert alert-danger">
+  <p>Currently, size variations can only be used in products in which size is a required field.</p>
+</div>
+
+<div class="alert alert-danger">
+  <p>
+    You cannot include HTML notation in <strong>any</strong> of the size variation's
```

Comment:
```suggestion
    Do not include HTML notation in <strong>any</strong> of the size variation's
```

Reply:
I see that we use it as "You cannot" in product's attributes  as well so ignore this.